src/molecules/sidebar/index.tsx

```diff
@@ -49,15 +49,28 @@ export const Sidebar = ({
 
   return (
     <div>
-      <Drawer open={isOpen} onClose={onToggle}>
+      <Drawer
+        open={isOpen}
+        onClose={onToggle}
+        PaperProps={{
+          sx: {
+            backgroundColor: '#002884',
+            color: 'white',
+            '& .MuiDrawer-paper': {
+              backgroundColor: '#002884',
+              color: 'white',
+            },
+          },
+        }}
+      >
         <Box sx={{ width: 250 }} role="presentation">
           {config && (
             <List>
               {config.showLangSwitcher && (
                 <ListItem disablePadding>
                   <ListItemButton onClick={handleItemClick}>
                     <ListItemIcon>
-                      <ArrowBackIcon />
+                      <ArrowBackIcon sx={{color: "white"}}/>
```

Comment:
Don't hard code the color to white here as well as for the other places too.

Use Themepicker for it and check some example usage  for it in other molecules

Reply:
ok I will see to it